resources/openebs-operator.yaml

```diff
@@ -213,7 +213,7 @@ metadata:
   labels:
     name: maya-apiserver
     openebs.io/component-name: maya-apiserver
-    openebs.io/version: 1.4.0
+    openebs.io/version: 1.5.0
```

Comment:
Why don't we use helm to template these things?

Reply:
Yeah, we should. At the moment we don't use Helm for our system apps b/c some customers disable Tiller for security concerns. Now that Helm 3 is out, we should upgrade to it and ideally rewrite all our system apps to be Helm charts. Then we'll be able to use templating.